flixel/FlxObject.hx

```diff
@@ -86,6 +86,11 @@ class FlxObject extends FlxBasic
 	 */
 	public static var SEPARATE_BIAS:Float = 4;
 
+	/**
+	 * The default `moves` value of all future `FlxObjects`
+	 */
+	public static var defaultMoves:Bool = true;
+
```

Comment:
```suggestion
	/**
	 * The default `moves` value of all future `FlxObjects`
	 * @since 5.6.0
	 */
	public static var defaultMoves:Bool = true;

```